english/make-friends/where_am_i

```diff
@@ -0,0 +1,16 @@
+You realise you need as much information as you need to get out of this situation.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.